libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.h

```diff
@@ -98,4 +101,30 @@
  @param animated Indicates whether or not the hiding should be animated.
  */
 - (void)hideHostListView:(BOOL)animated;
+
+/** Factory Method to create the back button.
+ */
+- (UIBarButtonItem *_Nonnull)createBackButton;
```

Comment:
Why do the create*Button methods need to be public?

Reply:
@wmathurin The create... methods should allow developers to customize the display of  3 items back button, setup button and the the Title. for e.g. the title could be a clickable item other than just a simple text.